docs/faq.md

```diff
@@ -181,31 +181,16 @@ You should not use prebundleSvelteLibraries during build or for ssr, disable one
 
 You really shouldn't. Svelte and Vite are esm first and the ecosystem is moving away from commonjs and so should you. Consider migrating to esm.
 
-In case you have to, use dynamic import to load vite-plugin-svelte's esm code from cjs like this:
+In case you have to, you can rely on node's "require esm" feature available in v20.19+
```

Comment:
```suggestion
In case you have to, you can rely on node's "require esm" feature available in v20.17+
```
According to https://nodejs.org/api/modules.html#loading-ecmascript-modules-using-require this was also backported to v20.17.0. Not sure if it's also in v20.18.0 though since that's not explicitly mentioned

Reply:
in 20.17 the feature was behind a flag, it was 20.19 that enabled it by default: https://nodejs.org/en/blog/release/v20.19.0

Reply:
Ah right. I misunderstood. My bad.